### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD003.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#7] [General]
# https://blog.3place.cn/3place/fsd/FSD003.html

https://blog.3place.cn/3place/fsd/FSD003.html

**Comment by picklecai** [comment]
她的步声像雷鸣一样。
天使见到她，获得生力，
虽然无人能究其根源。

原来人人都认识到太阳是我们这个世界里最强大的能源。

**Reply by hbaolong** [reply]
太阳是能量的源泉，代表父亲

**Comment by picklecai** [comment]
大海从海底岩石深处
奋然汹涌出洪涛万顷，
大海和岩石又被卷入
永远迅速的天体运行。

张力好足！

**Comment by picklecai** [comment]
世间永没有一事使你称心？

原来首先不称心的人不是浮士德，而是梅菲斯特。
这位梅菲斯特如此具有人味，但大部分人在他眼里仍然只是虫子。

**Comment by picklecai** [comment]
他好高骛远，心血沸腾，
他也有一半知道自己是笨伯；
他想摘下天上最美的星辰，
他想获得人间最大的快乐。

我们人类就永远处在这样一种能力与期望不匹配的矛盾之中。

**Comment by picklecai** [comment]
人在奋斗时，难免迷误。

前进中的一些弯路，就连上帝也能原谅。

**Comment by picklecai** [comment]
今天的内容，能看出来两条路有什么不同吗？
